rook/deploy-rook.sh

```diff
@@ -14,6 +14,9 @@ chart_version="v1.10.5"
 # Install rook operator
 kubectl create namespace "${namespace}" --dry-run -o yaml | kubectl apply -f -
 kubectl label namespace "${namespace}" owner=operator --overwrite
+kubectl label namespace "${namespace}" pod-security.kubernetes.io/audit=privileged --overwrite
```

Comment:
I do not think you need the overwrite?
```suggestion
kubectl label namespace "${namespace}" pod-security.kubernetes.io/audit=privileged
```